app/views/live_issues/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<h2>
+  Manage live issues
+</h2>
+
+<p>
+  Use this page to manage live issues with unresolved data affecting data outputs, workflows, or business rules.
+</p>
+
+<%= link_to 'Add a Live Issue', new_live_issue_path, class: 'govuk-button' %>
+
+<table class="govuk-table">
```

Comment:
Should we just use govuk components govuk_table helper like the dev-hub?

```ruby
    <%= govuk_table do |table|
          table.with_body do |body|
            body.with_row do |row|
              row.with_cell(header: true, text: 'API Key')
              row.with_cell(text: mask_api_key(@api_key))
            end
          end
        table.with_body do |body|
          body.with_row do |row|
            row.with_cell(header: true, text: 'Description')
            row.with_cell(text: @api_key.description)
          end
          body.with_row do |row|
            row.with_cell(header: true, text: 'Created on')
            row.with_cell(text: creation_date(@api_key))
          end
        end
      end
    %>
```

Reply:
I did think about that but I'm trying to keep it uniform with what we have in frontend atm. I'm not sure if myOTT has used the govuk components as well?